apps/web/src/pages/TalentNominations/NominationGroupEvaluationDialog/components/NominationGroupEvaluationForm.tsx

```diff
@@ -0,0 +1,195 @@
+    NominationGroupEvaluationForm_Fragment,
+    talentNominationGroupQuery,
+  );
+  const talentNominationGroupOptions = getFragment(
+    NominationGroupEvaluationFormOptions_Fragment,
+    talentNominationGroupOptionsQuery,
+  );
+
+  const methods = useForm<FormValues>({
+    defaultValues: convertQueryDataToFormData(talentNominationGroup),
+  });
+  const { handleSubmit } = methods;
+
+  const isNominatedForAdvancement =
+    talentNominationGroup?.advancementNominationCount;
+  const isNominatedForLateralMovement =
+    talentNominationGroup?.lateralMovementNominationCount;
+  const isNominatedForDevelopmentPrograms =
+    talentNominationGroup?.developmentProgramsNominationCount;
+
+  return (
+    <FormProvider {...methods}>
+      <form onSubmit={handleSubmit(onSubmit)}>
+        <div
+          data-h2-display="base(flex)"
+          data-h2-flex-direction="base(column)"
+          data-h2-gap="base(x1.25)"
+        >
+          <div
+            data-h2-display="base(flex)"
+            data-h2-flex-direction="base(column)"
+            data-h2-gap="base(x0.5)"
+          >
+            <p>
+              {intl.formatMessage({
+                defaultMessage:
+                  "Ready to submit the evaluation of this nomination? Please complete the form for the selected nomination options.",
+                id: "eGlL1Q",
+                description:
+                  "Introduction for form to evaluate a nomination group",
+              })}
+            </p>
+            <p>
+              {intl.formatMessage(
+                {
+                  defaultMessage: "{nomineeName} has been nominated for: ",
+                  id: "OJPiPD",
+                  description:
+                    "Introduction for list of nomination options for form to evaluate a nomination group",
+                },
+                {
+                  nomineeName: talentNominationGroup?.nominee?.firstName,
+                },
+              )}
+            </p>
+            <ul data-h2-margin-bottom="base:children[li:not(:last-child)](x.5)">
+              {isNominatedForAdvancement ? (
+                <li>
+                  {intl.formatMessage(
+                    talentNominationMessages.nominateForAdvancement,
+                  )}
+                </li>
+              ) : null}
+              {isNominatedForLateralMovement ? (
+                <li>
+                  {intl.formatMessage(
+                    talentNominationMessages.nominateForLateralMovement,
+                  )}
+                </li>
+              ) : null}
+              {isNominatedForDevelopmentPrograms ? (
+                <li>
+                  {intl.formatMessage(
+                    talentNominationMessages.nominateForDevelopmentPrograms,
+                  )}
+                </li>
+              ) : null}
+            </ul>
+          </div>
+          {isNominatedForAdvancement ? (
+            <>
+              <Separator data-h2-margin="base(0)" />
```

Comment:
I think these probably should have `decorative` it is very rare that our use of separators is not for decoration

Reply:
Oh, you're right.  We might consider making that the default for the component.
[mark separators as decorative](https://github.com/GCTC-NTGC/gc-digital-talent/pull/13305/commits/8f2ea69920672a33511b9fa12b18c6ea91955410)

Reply:
Yeah, I have been considering that for a while. I have, surprisingly, never some across a proper separator in app. I think we use them more like borders than separators :thinking: